vars/default.yml

```diff
@@ -1,2 +1,5 @@
 ---
-certbot_package: certbot
+certbot_package:
```

Comment:
Are `certbot-apache` and `certbot-nginx` required for certbot functionality?

I'd rather leave it up to the end user to install extra addons if they want...